package.json

```diff
@@ -11,6 +11,7 @@
     "node": ">=5"
   },
   "scripts": {
+    "postinstall": "cd lighthouse-core && npm install && npm run build",
```

Comment:
oh, you only have `lighthouse-core/` here, but if we're going this route we should arguably do it for the other dirs as well